### PR TITLE
Add project url in details with install feature using git clone command

### PR DESCRIPTION
**Pull request by FantacciSimone**
ref: https://llllllll.co/t/folio/47053/4?u=doberman

### CODE REVIEW

File: folio.lua [hunk 1]
```diff
@@ -163,11 +166,30 @@ function key(n, z)
...
+	          print(url_name)
+		  pull(url_name)
+	        end)
+        redraw()
       end
     end
   end
 end
 
+function pull(url)
```
Comment (csboling) [inline folio.lua:181]: Do we need `mkdir` and `sudo`? I would think `git clone` will create the target directory.

File: folio.lua [hunk 2]
```diff
@@ -163,11 +166,30 @@ function key(n, z)
         end
         page = 'scripts'
         redraw()
+      elseif s:sub(1, #'url: ') == 'url: ' then
+        local url_name = s:sub(#'url: ' + 1, #s)
+            clock.run(function()
```
Comment (csboling) [inline folio.lua:171]: Some extra indentation here.

File: folio.lua (hunk 1 above)
Comment (FantacciSimone) [reply folio.lua:181]: a first test without creating the folder manually did not work. Later I also had to add administrator access ... so maybe using "sudo" you don't need to create the folder manually ... did you do a test?

**Comment by csboling** [comment]
Thanks for the patch, I tweaked this a bit in https://github.com/csboling/folio/commit/a90ebb977a7b8383a11156e6e0f671aebbd23b31 so it will show "download" or "launch" depending on whether you already have the script installed, and "launch" will attempt to load the selected script. I did not have to `mkdir` the destination directory first or use `sudo`.

**Pull request merged**